sdks/python/.yapfignore

```diff
@@ -21,7 +21,7 @@ apache_beam/runners/dataflow/internal/clients/dataflow/dataflow_v1b3_messages.py
 apache_beam/io/gcp/internal/clients/storage/storage_v1_client.py
 apache_beam/io/gcp/internal/clients/storage/storage_v1_messages.py
 apache_beam/coders/proto2_coder_test_messages_pb2.py
-apache_beam/portability/api/*pb2*.py
+apache_beam/portability/api/*
```

Comment:
Can you confirm that we have only generated code in this subdirectory? (Does the ** pattern of "match all subdirectories not work?)

Reply:
I think when I tried, it did not work. i updated it so let's see what CI does

Reply:
https://ci-beam.apache.org/job/beam_PreCommit_PythonFormatter_Commit/10527/console

```
12:36:08 py3-yapf-check run-test: commands[1] | time yapf --diff --parallel --recursive apache_beam
12:36:35 --- apache_beam/portability/api/__init__.py	(original)
12:36:35 +++ apache_beam/portability/api/__init__.py	(reformatted)
```

I think I will need to revert, I can confirm all the files in that directory are generated